docs/pages/tutorial/eas/team-development.mdx

```diff
@@ -121,11 +121,11 @@ We'll modify the text of the first button in our example app that says **Choose
 
 Instead of creating a new build to share this change with our team for testing, let's publish an update:
 
-<Terminal cmd={['$ eas update --branch development --message "Change first button label"']} />
+<Terminal cmd={['$ eas update --channel development --message "Change first button label"']} />
```

Comment:
hiya @jonsamp, is it okay to update this command to use `channel` instead of `branch` since elsewhere in docs we are now recommending to use `channel` over `branch`?